packages/protect/package.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
@ancheetah Can we mark `sideEffects` in this package.json. I believe we have to mark the signals sdk as a side effect like we do in legacy.